pkg/commands/export/export.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package export
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/commands/cmdlib"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/converter"
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/files"
+	log "github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// options represents options flags for the export command.
+type options struct{}
+
+// opts is a global options instance for reference via the add commands.
+var opts = &options{}
+
+func action(ctx context.Context, cmd *cobra.Command, _ []string) {
+	gopt := cmdlib.GlobalOptionsValues.Copy()
+	stdio := cmdlib.RealStdioReaderWriter{}
+	brw := cmdlib.NewBundleReaderWriter(
+		&files.LocalFileSystemReaderWriter{},
+		&stdio,
+	)
+	if err := run(ctx, opts, brw, stdio, gopt); err != nil {
+		log.Exit(err)
+	}
+}
+
+func run(ctx context.Context, o *options, brw cmdlib.BundleReaderWriter, stdio cmdlib.RealStdioReaderWriter, gopt *cmdlib.GlobalOptions) error {
```

Comment:
The only thing I'd like to see added before this code is merged, and I would highly appreciate, is just some example input files and the expected output.   This could be files of the expected input vs output, a formal test, just some comments inline here that show the form -- whatever works for you.

Reply:
Take a look at the example I just added. It doesn't show the output directly but does show the commands.